css/styles.css

```diff
@@ -192,12 +254,58 @@ body {
   text-transform: uppercase;
 }
 
+/*.input--submit:hover {
+  background-color: thistle;
+}*/
+
+.input--submit-answers { 
+  background-color: red;
+  color: white;
+}
+
+.input--submit-answers:hover {
+  background-color: green;
+}
+
 .input--submit-names {
+  grid-column: 2/3;
   height: 36px;
-  width: 440px;
+  width: 200px;
+  opacity: .8;
+  justify-self: center;
   border-radius: 18px;
-  background-color: #2e1d75;
-  color: #ffffff;
+  color: thistle;
+  margin-top: 10px;
+}
+
+.button--exit {
+  background-color: red;
+}
+
+.button--exit:hover {
+  background-color: green;
+}
+
+.input--submit-names:hover {
+  opacity: 1;
+  color: lightskyblue;
+}
+
```

Comment:
These are the multiple choice answers for the user to pick from.  The red background makes it a little tough to read the answers but i dont think its a make or break. When hovering, they change to green (going with the vegas theme) and make the answers more prominent.